README.md

```diff
@@ -213,6 +213,22 @@ export default Eng;
 Currently, only services and route paths (see below) can be shared across the
 parent/engine boundary.
 
+If you want to share components, helpers or utils between a parent and
+an engine, then create an addon and add it as a dependency of both
+your application and engine.
+
+In-repo addons can be consumed from an engine. To do so, add it to the
+engine's `package.json`:
+
+```
+// lib/my-in-repo-engine/package.json
+  "ember-addon": {
+    "paths": [
+      "../shared-components"
```

Comment:
@nathanhammond makes sense, the point I want to make is that is still possible to consume in-repo addons, if you are using in-repo engines, maybe I should make that clear?